lxqtleave/translations/lxqt-hibernate_it.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Hibernate
+GenericName=Hibernate
+Comment=hibernate the machine
+Exec=lxqt-leave --hibernate
+Icon=system-suspend-hibernate
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
```

Comment:
The final \<app\>.desktop file is generated as concatenation of \<app\>.desktop.in file and translations/\<app\>\_\*.desktop file(s) (only \*\[\*\] keys are filtered). So there is no need for `translations` files to list all the universal keys.